blueprints/ember-frost-lts/index.js

```diff
@@ -1,4 +1,5 @@
 'use strict'
```

Comment:
Can you please check the full content of this file? I don't know what happened with my previous PR. I got merging issues.